src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/GeneralBrowseObject.xaml

```diff
@@ -26,11 +26,11 @@
       <DataSource PersistedName="RootNamespace" Persistence="ProjectFile" Label="Configuration" HasConfigurationCondition="false" />
     </StringProperty.DataSource>
   </StringProperty>
-  <StringProperty Name="TargetFramework" Visible="False">
-    <StringProperty.DataSource>
-      <DataSource Persistence="ProjectFile" PersistedName="TargetFrameworkMoniker" />
-    </StringProperty.DataSource>
-  </StringProperty>
+  <IntProperty Name="TargetFramework" Visible="False">
+    <IntProperty.DataSource>
+      <DataSource Persistence="ProjectFileWithInterception" PersistedName="TargetFramework" />
+    </IntProperty.DataSource>
```

Comment:
I think we'll want to make this read-only as well. This should never be persisted.